.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+| WARNING: Please, read this note carefully before submitting a new pull request |
+|--------------------------------------------------------------------------------|
+
+Although this software is open source and we welcome contributions,
+we believe these contributions should be preceded by an open discussion. 
+Open discussions tend to result in better solutions to any given problem,
+and help maintain and improve the quality of the software.
+If you would like to see a bug fixed or a new feature implemented,
+please open an issue for the discussion rather than directly opening
+a pull request.
+
+When you finally create a pull request, please follow CONTRIBUTING.md these and guidelines:
```

Comment:
If you would like to contribute please read [CONTRIBUTING.md](CONTRIBUTING.md).

When you finally create a pull request, please follow these guidelines: